eureka-client/src/main/java/com/netflix/appinfo/EurekaInstanceConfig.java

```diff
@@ -221,6 +221,13 @@
      */
     String getIpAddress();
 
+
+    /**
+     * Indicates if the public ipv4 address of the instance should be advertised.
+     * @return true if the public ipv4 address of the instance should be advertised, false otherwise .
+     */
+    boolean shouldBroadcastPublicIpv4Addr();
```

Comment:
Does this need to go in a public api? Looks like it could be in PropertiesInstanceConfig and used by CloudInstanceConfig without being in the interface.

Reply:
Yes, I was planning to throw in a default implementation as I realized it breaks the interface, but I could keep it to `PropertiesInstanceConfig` which was my initial plan. LMK what you prefer.

Reply:
@spencergibb PTAL

Reply:
What does ptal mean?

Reply:
Please Take A(nother) Look